### PR TITLE
[MU3] Fulfilled the default style file field in preferences with Leland.mss

### PR DESCRIPTION
**Pull request by vpereverzev**
изображение

### CODE REVIEW

**Comment by Marr11317** [comment]
what about style for part?

**Comment by vpereverzev** [comment]
> what about style for part?

good question, actually

**Comment by RobFog** [comment]
By "fulfil" you probably mean "fill out"?

**Comment by vpereverzev** [comment]
@RobFog who knows :) ("fill out" much better actually)

**Comment by vpereverzev** [comment]
@Marr11317 also updated that one, TY

**Comment by Jojo-Schmitz** [comment]
That is not the solution IMHO. It should be applied if, and only if, being to so by that dialog that shws for Pre 3.6 scores, and should be the default settings for new scores,

File: mscore/prefsdialog.ui [hunk 1]
```diff
@@ -90,7 +90,7 @@
       <string>Preferences Tab Manager</string>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>3</number>
```
Comment (Jojo-Schmitz) [inline mscore/prefsdialog.ui:93]: bad!!!

**Comment by Jojo-Schmitz** [comment]
And please at least provide a 3.5 mss style file, as an option, next to the MuseJazz and Leland ones, like an Emmentaler.mss

**Pull request merged**